src/Public/FavIcon.ps1

```diff
@@ -0,0 +1,346 @@
+<#
+.SYNOPSIS
+	Adds a favicon to one or more Pode HTTP/HTTPS endpoints.
+
+.DESCRIPTION
+	This function allows you to define a favicon for Pode endpoints. You can either use the default favicon provided
+	by Pode, specify a file path to a `.ico` file, or directly supply the favicon as a byte array. The function can apply
+	the favicon to all HTTP/S endpoints, a specific endpoint, or only to endpoints marked as default.
+
+.PARAMETER Default
+	Use the default Pode favicon.ico embedded in the module.
+
+.PARAMETER Path
+	The path to a custom favicon.ico file to use. Must be a valid, accessible path.
+
+.PARAMETER Binary
+	A raw byte array representing the favicon.ico file contents.
+
+.PARAMETER EndpointName
+	The name of the specific endpoint to apply the favicon to. If not provided, the favicon is applied to all HTTP/S endpoints.
+
+.PARAMETER DefaultEndpoint
+	If supplied, only endpoints marked with `.Default = $true` will receive the favicon.
```

Comment:
For each of the `DefaultEndpoint` parameter descriptions, I would start it with the following instead:

```
If supplied, only endpoints created with `-Default` ...
```

As it's more what the user will see in their code, rather than the internal `.Default = $true`